script/Init.s.sol

```diff
@@ -14,7 +14,7 @@ import { Broker, LockupDynamic, LockupLinear } from "../src/types/DataTypes.sol"
 import { BaseScript } from "./Base.s.sol";
 
 interface IERC20Mint {
-    function mint(address beneficiary, uint256 amount) external;
+    function mint(address beneficiary, uint256 value) external;
```

Comment:
Why did you change the `amount` to `value` here?

Reply:
because OpenZeppelin does not say `amount` anymore but `value` for all function parameter names

so, i wanted to follow their approach